### PR TITLE
False positive security vulnerabilities

### PR DESCRIPTION
**Pull request by Hixie**
We occasionally get reports of security vulnerabilities. This document attempts to explain them so that we can just point people to this document instead of having to explain it each time.

### CODE REVIEW

**Comment by Hixie** [comment]
I can't figure out why `build / stage (pull_request)` is failing.

**Comment by parlough** [comment]
Sorry about that. I’ve submitted a fix in https://github.com/dart-lang/site-www/pull/4507 and am tracking not breaking the build on a release in https://github.com/dart-lang/site-www/issues/4297.

File: src/resources/security-false-positives.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
+---
+title: Security False Positives
```
Comment (mit-mit) [inline src/resources/security-false-positives.md:2]: nit: please use sentence casing

**Review by mit-mit — COMMENTED** [review]
A high-level question before I review the details: Should this perhaps be on flutter.dev, not dart.dev? Yes, there are quite a few Dart details, but most of the content is about the Dart embedding in Flutter, not Dart itself. For example, Dart per se doesn't have a `libapp.so`, that's a Flutter thing.

**Comment by mit-mit** [comment]
cc @mraleph

**Comment by mraleph** [comment]
Yeah, this probably should be on `flutter.dev`. I did review the original doc so I am not sure I have anything else to say :)

**Comment by Hixie** [comment]
I'll let @sfshaza2 and y'all debate which site it should be on. I have no strong opinion. Let me know when you come to a conclusion. :-)

**Comment by sfshaza2** [comment]
I thought it should go on dart.dev because it was primarily dart-focused. But if these errors occur mostly with Flutter apps, then, yes, let's do that. Sorry, Ian!  We can put it under src/testing

**Review by sfshaza2 — COMMENTED** [review]
Closing this PR because I'm putting it on the docs.flutter.dev site.

**Pull request closed without merging**